Mathlib/Data/List/AList.lean

```diff
@@ -241,13 +241,15 @@ theorem insert_entries {a} {b : β a} {s : AList β} :
     (insert a b s).entries = Sigma.mk a b :: kerase a s.entries :=
```

Comment:
I guess for consistency we should also deprecate this one and call it `entries_insert`